crates/core/component/ibc/src/component/msg_handler/channel_open_init.rs

```diff
@@ -57,6 +57,7 @@ impl MsgHandler for MsgChannelOpenInit {
             remote: Counterparty::new(self.port_id_on_b.clone(), None),
             connection_hops: self.connection_hops_on_a.clone(),
             version: self.version_proposal.clone(),
+            ..ChannelEnd::default()
```

Comment:
`ChannelEnd` now has a default impl - so I used it for all new fields. Could set them explicitly?